src/libslic3r/libslic3r.h

```diff
@@ -55,7 +55,7 @@ static constexpr double EPSILON = 1e-4;
 // 0..4294mm with 1nm resolution
 // int32_t fits an interval of (-2147.48mm, +2147.48mm)
 // with int64_t we don't have to worry anymore about the size of the int.
-static constexpr double SCALING_FACTOR = 0.000001;
```

Comment:
The comments above this need updating, since the range of representable values changes here.